### DISCUSSION TITLE
Hallypop

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#654] [Q&A]
**Channel Name:** GMA Hallypop/Hallypop Philippines
**Channel ID:** Hallypop.ph
**Country:** Philippines
**Language:** English/Korean
**External Link (optional):** xxx
**Notes (optional):** Free to air local version of Hallypop in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19654
